hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java

```diff
@@ -161,4 +163,31 @@ public static void activate() {
     Configuration.addDefaultResource("ozone-default.xml");
     Configuration.addDefaultResource("ozone-site.xml");
   }
+
+  /**
+   * The super class method getAllPropertiesByTag
```

Comment:
Do we need an other (HADOOP) jira to fix it on the hadoop 3.3 line, too?

Reply:
Yes @elek